extensions/wikia/Chat2/js/ChatWidget.js

```diff
@@ -16,13 +16,17 @@ var ChatWidget = {
 		}
 
 		// make sure we start processing after ChatModule templates is loaded
-		if ($('.chat-module').length) {
+		// SUS-1202: refresh user list for in-content chat widget, the one on the right rail is always loaded fresh
+		if ($('.chat-module').not('.rail-module').length) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Merge this if statement with the nested one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3ACollapsibleIfStatements)